src/js/views/Project/Create/Attributes.jsx

```diff
@@ -4,7 +4,7 @@ import { default as slugify } from 'slugify'
 import { useTranslation } from 'react-i18next'
 import { validate } from 'jsonschema'
 
-import { asOptions } from '../../../metadata'
+import { metadataAsOptions } from '../../../settings'
```

Comment:
You missed a similar change in [views/Projects/Projects.jsx](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Projects/Projects.jsx#L2)